actors/system/src/lib.rs

```diff
@@ -25,9 +25,25 @@ pub enum Method {
 /// System actor state.
 #[derive(Default, Deserialize, Serialize)]
 #[serde(transparent)]
-pub struct State([(); 0]);
+pub struct State {
+    // builtin actor registry: Vec<(String, Cid)>
+    builtin_actors: Cid,
+}
 impl Cbor for State {}
 
+impl State {
+    pub fn get_builtin_actors<B: Blockstore>(
+        &self,
+        store: &B,
```

Comment:
```suggestion
    pub fn get_builtin_actors(
        &self,
        store: &impl Blockstore,
```

Reply:
(just slightly easier to read)

Reply:
ok.

Reply:
note that we don't do that anywhere else, so inconsistent with the rest of the code.

I think I'll keep it as is (although I agree that the impl spell is nicer), for consistency and obviousness of the generic.

Reply:
Yeah, consistency wins. If we want to change, we can do a mass change.